man/systemd.exec.xml

```diff
@@ -2675,6 +2669,11 @@ StandardInputData=SWNrIHNpdHplIGRhIHVuJyBlc3NlIEtsb3BzLAp1ZmYgZWVtYWwga2xvcHAncy
                 <entry>not set</entry>
                 <entry>not set</entry>
               </row>
+              <row>
+                <entry valign="top"><literal>exec-condition</literal></entry>
```

Comment:
"exec-condition" should only have `$EXIT_CODE` "exited" and `$EXIT_STATUS` "1 ... 254" as a row (like in your original change). Because if it were to hit one of the other terminated conditions (255, SIGTERM, core dump, etc), the `$SERVICE_RESULT` would no longer be "exec-condition". Apologies since I think I said something misleading offline!